lib/src/chip/filter-button/filter-button.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, HostBinding, Input, OnChanges } from '@angular/core';
+
+@Component({
+    selector: 'hc-filter-button',
+    template: `<img src="./assets/FilterIcon.svg" class="filter-img">{{buttonLabel}}`,
+    styleUrls: ['../chip.component.scss']
+})
+export class FilterButtonComponent implements OnChanges  {
+    @HostBinding('class') hostClass = 'filter-button';
```

Comment:
This could wipe out a user's class that they define on the element. Go with
`@HostBinding('class.filter-button') hostClass = true;`